packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -373,3 +374,58 @@ test.describe('`isDismissible` prop', () => {
         });
     });
 });
+
+test.describe('isOpen prop', () => {
+    test('should not render open when isOpen = false', async ({ mount, page }) => {
+        // Arrange
+        await mount(PieModal, {
+            props: {
+                isOpen: false,
+            },
+        });
+
+        // Assert
+        await expect(page.locator(modalSelector)).not.toBeVisible();
+    });
+
+    test('should render open when isOpen = true', async ({ mount, page }) => {
+        // Arrange
+        await mount(PieModal, {
+            props: {
+                isOpen: true,
+            },
+        });
+
+        // Assert
+        await expect(page.locator(modalSelector)).toBeVisible();
+    });
+});
+
+test('Should not be able to scroll when modal is open', async ({ page, mount }) => {
+    // Arrange
+    const modalComponent = renderTestPieModal();
+
+    await mount(
+        WebComponentTestWrapper,
+        {
+            props: {
+                pageMode: true,
+            },
+            slots: {
+                component: modalComponent,
+                pageMarkup: createScrollablePageHTML(),
+            },
+        },
+    );
+
+    // Act
+    // Scroll 800 pixels down the page
+    await page.mouse.wheel(0, 1000);
+
+    // The mouse.wheel function causes scrolling, but doesn't wait for the scroll to finish before returning.
+    await page.waitForTimeout(3000);
+
+    // Assert
+    await expect.soft(page.getByText('Test Page')).toBeInViewport();
+    await expect(page.getByText('Test Item')).not.toBeInViewport();
```

Comment:
I think we can make the items a bit more descriptive here. Ideally I shouldn't need to check the other file to understand what test page and test item are doing.

Even renaming them to something like `top of page copy` and `bottom of page copy` might be useful